proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -131,11 +122,14 @@ message Dispatch {
   // The creation time
   google.protobuf.Timestamp create_time = 7;
 
-  // The dispatch status
-  DispatchStatus status = 8;
+  // The "active" status
+  bool active = 8;
+
+  // The "dry run" status
+  bool dry_run = 9;
```

Comment:
Since these are boolean flags now, then maybe `is_active` and `is_dry_run` are more conventional here? Not a strong opinion though.